docs/DeveloperGuide.md

```diff
@@ -1,13 +1,39 @@
 # Developer Guide
```

Comment:
It could be helpful to include a section for Introduction, to ease new developers into the project more seamlessly.